ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml

```diff
@@ -166,26 +171,24 @@ ModalPopup {
         }
     }
 
-    footer: Button {
-        width: 44
-        height: 44
+    footer: Item {
```

Comment:
I wonder if we could just use StatusButton directly here

Reply:
Yes we can. Also, only the `anchor.right` is necessary

Reply:
Yes, thats a much cleaner approach. I've added both this changes!